openreview/conference/templates/areachairWebfield.js

```diff
@@ -651,12 +675,89 @@ var registerEventHandlers = function() {
   });
 
   $('#group-container').on('click', 'a.collapse-btn', function(e) {
-    $(this).next().slideToggle();
     if ($(this).text() === 'Show reviewers') {
       $(this).text('Hide reviewers');
     } else {
       $(this).text('Show reviewers');
     }
+  });
+
+  $('#group-container').on('click', 'button.btn.btn-assign-reviewer', function(e) {
+    var $link = $(this);
+    var paperNumber = $link.data('paperNumber');
+    var paperForum = $link.data('paperForum');
+    var $currDiv = $('#' + paperForum + '-add-reviewer');
+    userToAdd = $currDiv.find('input').val().trim();
```

Comment:
Is this var defined somewhere else or is it missing the `var`?

Reply:
It was missing the var. Corrected.